### PR TITLE
Add kube-scheduler config v1 to reference index

### PR DESCRIPTION
**Pull request by alculquicondor**
v1 is available in 1.25.

### CODE REVIEW

File: content/en/docs/reference/_index.md [hunk 1]
```diff
@@ -82,8 +82,7 @@ operator to use or manage a cluster.
   [kubelet configuration (v1beta1)](/docs/reference/config-api/kubelet-config.v1beta1/)
 * [kubelet credential providers (v1alpha1)](/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)
 * [kubelet credential providers (v1beta1)](/docs/reference/config-api/kubelet-credentialprovider.v1beta1/)
-* [kube-scheduler configuration (v1beta2)](/docs/reference/config-api/kube-scheduler-config.v1beta2/) and
-  [kube-scheduler configuration (v1beta3)](/docs/reference/config-api/kube-scheduler-config.v1beta3/)
+* [kube-scheduler configuration (v1beta2)](/docs/reference/config-api/kube-scheduler-config.v1beta2/), [kube-scheduler configuration (v1beta3)](/docs/reference/config-api/kube-scheduler-config.v1beta3/) and [kube-scheduler configuration (v1)](/docs/reference/config-api/kube-scheduler-config.v1/)
```
Comment (tengqm) [inline content/en/docs/reference/_index.md:85]: please wrap the long lines as we did  before.

```suggestion
* [kube-scheduler configuration (v1beta2)](/docs/reference/config-api/kube-scheduler-config.v1beta2/),
  [kube-scheduler configuration (v1beta3)](/docs/reference/config-api/kube-scheduler-config.v1beta3/) and
  [kube-scheduler configuration (v1)](/docs/reference/config-api/kube-scheduler-config.v1/)
```

**Comment by sftim** [comment]
/approve

Tweak needed but the intent is 100% valid.

File: content/en/docs/reference/_index.md (hunk 1 above)
Comment (alculquicondor) [reply content/en/docs/reference/_index.md]: Done

**Comment by sftim** [comment]
Thanks @alculquicondor

**Pull request merged**